packages-ts/gauntlet-terra-cw20-multisig/package.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "@chainlink/gauntlet-terra-cw20-multisig",
```

Let' s rename this package and have it support the cw-plus contracts we need:

- cw3-flex-multisig
- cw4-group
- cw20-base

```suggestion
  "name": "@chainlink/gauntlet-terra-cw-plus",
```